docs/user/security/authentication/index.asciidoc

```diff
@@ -46,7 +47,7 @@ xpack.security.authc.providers: [token, basic]
 --------------------------------------------------------------------------------
 
 [[pki-authentication]]
-==== Public Key Infrastructure (PKI) Authentication
+==== Public key infrastructure (PKI) authentication
```

Comment:
Not sure if Public key infrastructure should be lower case.

Reply:
I think it's okay, since it's used that way other places (e.g. https://en.wikipedia.org/wiki/Public_key_infrastructure)